lib/evm/instruction/impl.ex

```diff
@@ -9,6 +9,7 @@ defmodule EVM.Instruction.Impl do
   alias EVM.SubState
```

Comment:
I 100% agree; it'll be nice to have a smaller set of more meaningful files that aren't thousands of lines long.